ide/views.py

```diff
@@ -82,6 +83,78 @@ def calculate_parameter(request):
         return JsonResponse({'result': 'success', 'net': net})
 
 
+@csrf_exempt
+def deleteModel_from_db(request):
+    if request.method == 'POST':
+        if 'userID' in request.POST:
+            userID = request.POST.get('userID')
+            model_id = request.POST.get('modelid')
+            model = Network.objects.get(id=model_id)
+            if model.author_id == int(userID):
+                model.delete()
+                return JsonResponse({'result': 'success', 'data': 'Model successfully deleted!'})
+            else:
+                return JsonResponse({'result': 'error', 'error': "This model doesn't belong to you!"})
+
+
+@csrf_exempt
+def saveModel_to_db(request):
```

Comment:
Why do we need to save model to db methods?

Reply:
How will I check using model id? If a person is creating a new model with the same model name then only I can run a query right?

Reply:
The save button triggers this function